poc/frost.sage

```diff
@@ -263,7 +263,7 @@ class SignerParticipant(object):
 
 # Configure the setting
 MAX_PARTICIPANTS = 3
-MIN_PARTICIPANTS = 2
+MIN_PARTICIPANTS = 1
```

Comment:
```suggestion
MIN_PARTICIPANTS = 2
```

I don't think this is what we want, here it's specifying the threshold for generating the test vectors, and not the minimum limit for all thresholds